### ISSUE TITLE
Failed to import - https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:2713

### ISSUE THREAD

**Issue by vixalien** [esm-dev/esm.sh#556]
## Failing module

- **GitHub**: https://github.com/websockets/ws
- **npm**: https://www.npmjs.com/package/ws

```js
// very complicated but basically
import * as ws from 'https://github.com/websockets/ws'
```

## Error message

After running `deno run` I got this:

```
error: Uncaught TypeError: __Buffer$.alloc is not a function
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:2713
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:969
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:2982
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:969
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:4355
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:969
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:11129
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:969
    at https://esm.sh/v111/ws@8.13.0/denonext/ws.js:8:182
```

## Additional info

I believe this is caused by incorrect importing of the `node:buffer` package.

In fact, `https://esm.sh/v111/ws@8.13.0/denonext/ws.js:2:2713` links to the following source line:

https://github.com/websockets/ws/blob/45e17acea791d865df6b255a55182e9c42e5877a/lib/constants.js#L5

This gets transpired to:

```ts
import __Buffer$ from "node:buffer";
//...
EMPTY_BUFFER:__Buffer$.alloc(0)
//...
```

which is incorrect, as the correct import should be `import { Buffer } from "node:buffer" `.

In short, it imports the default export instead of the named export `Buffer`

- **esm.sh version**: 111
- **Deno version**: deno 1.31.3

**Comment by ije** [comment]
thanks for reporting, i just fixed. please use `deno` target for now, i will release the change soon

**Comment by ije** [comment]
please wait v112
